Lib/csv.py

```diff
@@ -117,7 +117,7 @@ def __next__(self):
         # values
         while row == []:
             row = next(self.reader)
-        d = OrderedDict(zip(self.fieldnames, row))
+        d = tuple(zip(self.fieldnames, row))
```

Comment:
Tests failed due to using tuple instead of list (as line 127 will attempt to write to what is now non-mutable)
Needs to be changed to list(zip...)